indico/cli/i18n.py

```diff
@@ -335,6 +335,64 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file, *input_files):
+    """Merge multiple POT files into a single POT file."""
+    merged_catalog = None
+
+    for input_file in input_files:
+        with input_file.open('rb') as f:
+            catalog = read_po(f)
+
+            if merged_catalog is None:
+                merged_catalog = catalog
+            else:
+                for message in catalog:
+                    if message.id not in merged_catalog:
+                        merged_catalog[message.id] = message
+
+    if merged_catalog:
+        with output_file.open('wb') as f:
+            write_po(f, merged_catalog)
+
+    print('\n✔  Done merging pot files!\n')
+
+
+# Filter merged.po files by removing messages that are not in the given .pot file
+def filter_po_by_pot(merged_po_path, pot_path, output_po_path):
+    with merged_po_path.open('rb') as f:
+        merged_catalog = read_po(f)
+
+    with pot_path.open('rb') as f:
+        pot_catalog = read_po(f)
+
+    pot_message_ids = {message.id for message in pot_catalog}
+
+    for message in list(merged_catalog):
+        if message.id not in pot_message_ids:
+            merged_catalog.delete(message.id)
```

Comment:
I _think_ this currently does not handle context:

```suggestion
            merged_catalog.delete(message.id, message.context)
```

There's also the `Catalog.update` method, which appears to handle the filtering automatically. Maybe we could use that?

Reply:
I tested it and you're right. It does work well with the `update` fn. Cool!

Reply:
This code is now different cause of the update function so the delete context is not relevant anymore.